xcell/tests/test_cls.py

```diff
@@ -382,6 +382,37 @@ def test_cov_spin0(perm):
             assert np.all(r[i::ncls1][1:] < 1E-5)
 
 
+def test_clfid_halomod_settings():
+    data = get_config()
+
+    # Empty halo model parameters (default values)
+    clf = ClFid(data, 'Dummy__0', 'Dummy__1')
+    assert np.fabs(clf.hm_par['mass_def'].get_Delta(clf.cosmo, 1.)
```

Comment:
What does `get_Delta` do?

Reply:
returns the spherical overdensity associated with this mass definition (200 in this case)